server/jetstream_cluster.go

```diff
@@ -4264,6 +4264,8 @@ func (js *jetStream) processClusterCreateConsumer(ca *consumerAssignment, state
 		} else {
 			// Check for scale down to 1..
 			if rg.node != nil && len(rg.Peers) == 1 {
+				// Need to pop loopAndForward by closing qch and nil out both qch and pch.
```

Comment:
Would be nice if we can expand this comment just to explain what happens if we don't, i.e. that it results in errant proposals.

Reply:
Updated this part